artemis-distribution/pom.xml

```diff
@@ -117,15 +117,12 @@
          <artifactId>artemis-console</artifactId>
          <type>war</type>
       </dependency>
+      <!-- needed for console NPM License-->
       <dependency>
          <groupId>org.apache.activemq</groupId>
-         <artifactId>activemq-branding</artifactId>
-         <type>war</type>
-      </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-plugin</artifactId>
-         <type>war</type>
+         <artifactId>apache-artemis-console</artifactId>
+         <type>zip</type>
+         <classifier>bin</classifier>
```

Comment:
The changes in the pom overall result in inclusion of the console zip and the war in the distribution lib dir. 

This should probably marked as provided as its only here for build time use (or maybe it could even be omitted, given it is specified in the dependency unpack execution)